ruby-challenges.rb

```diff
@@ -19,12 +43,53 @@
 nums_array2 = [25, 17, 47, 11]
 # Expected output: 100
 
+# PSUEDOCODE:
+# start at 0
+# for the length of the array, add each index to the next
+# method => .each ? or .sum ?
+#  return
+
+def summed_up array
+    # current_total = 0
+    array.sum
+end
+
+p summed_up(nums_array1) #result = 76
+p summed_up(nums_array2) #result = 100
+
 
 
 # --------------------3a) Create a class called Bike that is initialized with a model, wheels, and current_speed. The default number of wheels is 2. The current_speed should start at 0. Create a bike_info method that returns a sentence with all the data from the bike object.
 
 # Expected output example: 'The Trek bike has 2 wheels and is going 0 mph.'
 
+class Bike
+    def initialize(model, wheels, current_speed)
+        @model = model
+        @wheels = '2'
+        @current_speed = 0
+    end
+
+    def pedal_faster num
+        @current_speed += num
+    end
+    
+    def bike_info
+        "The #{@model} bike has #{@wheels} wheels and is traveling at #{@current_speed} mph."
+    end
+
+    # def slowing_down
+        
+    # end
+end
+
+p vroom = Bike.new('Huffy','2', 20).bike_info
```

Comment:
vroom...love the name!!! As things are setup right now, your vroom variable is currently storing that sentence "The Huffy bike has 2 wheels and is traveling at 20 mph." So when you try to use others methods it will not execute as expected. Reconsider how you are using that variable then you should be able to get the expected outputs.